### PR TITLE
DOCS-2316: Enclosure Management R Series Known issue

### PR DESCRIPTION
**Pull request by jervin777**
Added additional known issue regarding enclosure management and the R-Series.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/ReleaseNotes/CORE/12.0U4.md [hunk 1]
```diff
@@ -271,6 +271,10 @@ Please check out the new [TrueNAS documentation](https://www.truenas.com/docs/)
                 </thead>
                 <tbody>
                 	<tr>
+<td><a href="https://jira.ixsystems.com/browse/NAS-111220" target="_blank">NAS-111220</a></td>
+<td>Enclosure management mapping may not work properly for TrueNAS R-Series</td>
+<td>This issue will fixed with the 12.0-U5 Release.</td>
```
Comment (Mrt134) [inline content/ReleaseNotes/CORE/12.0U4.md:276]:
```suggestion
<td>This issue will be fixed with the 12.0-U5 Release.</td>
```

**Review by Mrt134 — COMMENTED** [review]
One grammar error, then good to go.

**Comment by jervin777** [comment]
Grammar fix committed.

**Comment by Mrt134** [comment]
Link checks are hitting on false positives in the gnupg website and TC api docs images. Approved for merge.

**Pull request merged**